docker/build-tools/Dockerfile

```diff
@@ -25,7 +25,7 @@
 # Binary tools
 ################
 
-ARG GOLANG_IMAGE=golang:1.16.4
+ARG GOLANG_IMAGE
```

Comment:
can we still set the default here but override it in the script? Then you don't need to run a complex script when a simple `docker build` works(?)

Reply:
Done, I have also updated  `build-tools-proxy` to be able to accept this build-arg, PTAL.